src/org/ses/android/soap/tasks/AgregarHuellaTask.java

```diff
@@ -0,0 +1,67 @@
+package org.ses.android.soap.tasks;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.ksoap2.SoapEnvelope;
+import org.ksoap2.serialization.SoapObject;
+import org.ksoap2.serialization.SoapPrimitive;
+import org.ksoap2.serialization.SoapSerializationEnvelope;
+import org.ksoap2.transport.HttpTransportSE;
+
+
+
+/**
+ * Created by franciscorivera on 1/8/16.
+ */
+public class AgregarHuellaTask extends AsyncTask<Byte, Void, String> {
```

Comment:
Could you explain in a comment here what this task does, including its parameters and output?